NodeManager.cpp

```diff
@@ -1318,6 +1398,66 @@ void SensorBH1750::onReceive(const MyMessage & message) {
 #endif
 
 /*
+   SensorTSL2561
+*/
+#if MODULE_TSL2561 == 1
+// contructor
+SensorTSL2561::SensorTSL2561(int child_id): Sensor(child_id,A4) {
+  setPresentation(S_LIGHT_LEVEL);
+  setType(V_LEVEL);
+  _tsl = new TLS2561(TSL2561_ADDR_FLOAT);
```

Comment:
Having another setter would be ideal but in this case the TLS2561 object has to be created somewhere else (e.g. onBefore or onSetup) because the constructor is called straight away by registerSensor() and the user would set this only later. 
Another thing we need to work around are those defines which are accessible in NodeManager.cpp but they are not in the main sketch. We can either ask the user to import the header file in the main sketch for this sensor or passing the address directly which is probably simpler.